packages/server/src/migrations/schema/v98.ts

```diff
@@ -0,0 +1,36 @@
+// SPDX-FileCopyrightText: Copyright Orangebot, Inc. and Medplum contributors
+// SPDX-License-Identifier: Apache-2.0
+/*
+ * This is a generated file
+ * Do not edit manually.
+ */
+
+import { PoolClient } from 'pg';
+import * as fns from '../migrate-functions';
+
+// prettier-ignore
+export async function run(client: PoolClient): Promise<void> {
+  const results: { name: string; durationMs: number }[] = []
+  await fns.query(client, results, `CREATE TABLE IF NOT EXISTS "ConceptMapping" (
+    "id" BIGSERIAL PRIMARY KEY,
```

Comment:
`BIGSERIAL/SERIAL` are basically legacy.  `IDENTITY` is the way go.

Should be: `"id" int8 GENERATED ALWAYS AS IDENTITY PRIMARY KEY` 

* https://wiki.postgresql.org/wiki/Don%27t_Do_This#Don.27t_use_serial
* https://www.enterprisedb.com/blog/postgresql-10-identity-columns-explained
* https://github.com/kristiandupont/schemalint/tree/main/src/rules#prefer-identity-to-serial